lib/zclient.c

```diff
@@ -1461,6 +1462,223 @@ zebra_interface_vrf_update_read (struct stream *s, vrf_id_t vrf_id,
   *new_vrf_id = new_id;
   return ifp;
 }
+/**
+ * Connect to label manager in a syncronous way
+ *
+ * It first writes the request to zcient output buffer and then
+ * immediately reads the answer from the input buffer.
+ *
+ * @param zclient Zclient used to connect to label manager (zebra)
+ * @result Result of response
+ */
+int
+lm_label_manager_connect (struct zclient *zclient)
```

Comment:
About the first one, good catch, thanks. It's fixed now.

About the second one, with that message we are setting proto and instance in zserv object, a little bit like ZEBRA_HELLO does. Maybe we could remove the release_daemon_chunks from the reception of that reception (in function [zread_label_manager_connect](https://github.com/bingen/frr/blob/label_manager_2/zebra/zserv.c#L1827)) but I don't think it hurts. If you think it's better to remove it, I'll do it.